### PR TITLE
[2.6.6] CIS-benchmark clusterName fix.

### PR DESCRIPTION
**Pull request by doflamingo721**
Linked issue:
https://github.com/rancher/cis-operator/issues/152

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/Chart.yaml [hunk 1]
```diff
@@ -12,11 +12,11 @@ annotations:
...
-appVersion: v2.0.4
+appVersion: v2.0.5
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
 keywords:
 - security
 name: rancher-cis-benchmark
-version: 2.0.4
+version: 2.0.5
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/Chart.yaml:22]: We should bump the chart to rc version before the release. We should 2.0.5-rc1 until we are ready for release

File: release.yaml [hunk 2]
```diff
@@ -12,3 +12,7 @@ rancher-gatekeeper:
 - 100.1.1+up3.7.1
 rancher-gatekeeper-crd:
 - 100.1.1+up3.7.1
+rancher-cis-benchmark:
+- 2.0.5
+rancher-cis-benchmark-crd:
+- 2.0.5
```
Comment (prachidamle) [inline release.yaml:18]: use rc tags

File: packages/rancher-cis-benchmark/charts/Chart.yaml (hunk 1 above)
Comment (doflamingo721) [reply packages/rancher-cis-benchmark/charts/Chart.yaml]: updated to rc version

File: release.yaml (hunk 2 above)
Comment (doflamingo721) [reply release.yaml]: updated to rc versions

**Review by prachidamle — DISMISSED** [review]
lgtm

**Review by prachidamle — APPROVED** [review]

**Pull request merged**